core/src/test/java/org/mobilitydata/gtfsvalidator/testgtfs/GtfsStop.java

```diff
@@ -16,13 +16,296 @@
 
 package org.mobilitydata.gtfsvalidator.testgtfs;
 
+import com.google.common.geometry.S2LatLng;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 import org.mobilitydata.gtfsvalidator.table.GtfsEntity;
+import org.mobilitydata.gtfsvalidator.table.GtfsEntityBuilder;
 
 /** Test class to avoid dependency on the real GtfsStop and annotation processor. */
 public class GtfsStop implements GtfsEntity {
```

Comment:
Why did you need to expand GtfsStop to contain all fields?

In general, I'd actually be in favor of renaming these `GtfsStop` test classes to something like `GtfsTestEntity` (with a minimal set of test fields) to make it clearer that they are not to be confused with the actual `GtfsStop` class.

Reply:
I expanded this class with some fields, not all of them, to be able to parse a valid file without having to mock a lot of methods. Too many mocks increase the efforts of maintaining the test code. I agree that the class name can be misleading, I'll change it to a more appropriate name like `GtfsTestEntity`. I'll also reduce the fields and code within the class.